src/components/Hero/Hero.scss

```diff
@@ -63,3 +66,21 @@
     border: var(--color-fill-action) var(--space-02) solid;
   }
 }
+
+@media (max-width: 720px) {
+  .home-page-hero {
+    padding: var(--space-96) 0 var(--space-96) 0;
+    h1 {
+      font-size: 40px;
```

Comment:
perhaps we could use some of the [Typography](https://github.com/nodejs/nodejs.dev/blob/staging/src/styles/tokens.scss#L93) based on [designs](https://www.figma.com/proto/lOxAGGg5KXb6nwie7zXkz6/NJ---Design-System)?